test/e2e/flowaggregator_test.go

```diff
@@ -264,20 +279,53 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 						}
 					}
 				}
-				if checkNetworkPolicy {
+				if checkK8sNetworkPolicy {
 					// Check if records have both ingress and egress network policies.
 					if !strings.Contains(record, ingressNetworkPolicyName) {
 						t.Errorf("Record does not have NetworkPolicy name with ingress rule")
 					}
 					if !strings.Contains(record, fmt.Sprintf("%s: %s", "ingressNetworkPolicyNamespace", testNamespace)) {
 						t.Errorf("Record does not have correct ingressNetworkPolicyNamespace")
 					}
+					if !strings.Contains(record, fmt.Sprintf("%s: %d", "ingressNetworkPolicyType", ipfixregistry.PolicyTypeK8sNetworkPolicy)) {
+						t.Errorf("Record does not have correct NetworkPolicyType with ingress rule")
+					}
 					if !strings.Contains(record, egressNetworkPolicyName) {
 						t.Errorf("Record does not have NetworkPolicy name with egress rule")
 					}
 					if !strings.Contains(record, fmt.Sprintf("%s: %s", "egressNetworkPolicyNamespace", testNamespace)) {
 						t.Errorf("Record does not have correct egressNetworkPolicyNamespace")
 					}
+					if !strings.Contains(record, fmt.Sprintf("%s: %d", "egressNetworkPolicyType", ipfixregistry.PolicyTypeK8sNetworkPolicy)) {
+						t.Errorf("Record does not have correct NetworkPolicyType with egress rule")
+					}
+				}
+				if checkAntreaNetworkPolicy {
+					// Check if records have both ingress and egress network policies.
+					if !strings.Contains(record, ingressAntreaNetworkPolicyName) {
+						t.Errorf("Record does not have NetworkPolicy name with ingress rule")
+					}
```

Comment:
can we use `assert.Contains` here for less verbosity? Including for the checks which pre-date this PR

Reply:
updated, thanks!